src/style.css

```
@@ -1,3 +1,88 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+}
+
+@media screen and (max-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+.img__logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+  margin-left: 50px;
+}
+
+nav {
+  display: inline-block;
+  margin-right: 50px;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-style: normal;
+  line-height: 60px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-right: 20px;
+  position: relative;
+  padding: 22px 0 23px;
```

```suggestion
  padding: 23px 0;
  display: flex;
```
redundant to set line-height on full height of header and vertical paddings
vertical paddings 23px with line height 14px and display flex or block will make height 60px